pkgs/development/compilers/adoptopenjdk-bin/jdk-linux-base.nix

```diff
@@ -4,16 +4,31 @@ sourcePerArch:
 , lib
 , fetchurl
 , autoPatchelfHook
+, makeWrapper
+# minimum dependencies
 , alsaLib
-, freetype
 , fontconfig
-, zlib
-, xorg
+, freetype
 , libffi
+, xorg
+, zlib
+# runtime dependencies
+, cups
+# runtime dependencies for GTK+ Look and Feel
+, enableGTK ? true
```

Comment:
I have seen other packages in nixpkgs call these kinds of flags eg `gtkSupport` - I wonder if there's an official style guide for this?

Reply:
@taku0 ^

Reply:
or enableGTKSupport. I don't think there is any style for that.